packages/salesforcedx-vscode-apex/src/commands/forceAnonApexExecute.ts

```diff
@@ -249,7 +255,7 @@ export class AnonApexLibraryExecuteExecutor extends LibraryCommandletExecutor<Ap
   }
 }
 
-export const forceAnonApexExecute = async () => {
+export const forceAnonApexExecute =  async () => {
```

Comment:
```suggestion
export const forceAnonApexExecute = async () => {
```
Extra space was added here.